simpleio.py

```diff
@@ -28,14 +28,23 @@
 * Author(s): Scott Shawcroft
 """
 import time
-import sys
-try:
-    import audioio
-except ImportError:
-    pass # not always supported by every board!
 import array
 import digitalio
 import pulseio
+try:
+    import audiocore
+except ImportError:
+    try:
+        import audioio as audiocore
+    except ImportError:
+        pass # not always supported by every board!
+try:
+    from audioio import AudioOut
+except ImportError:
+    try:
+        from audiopwmio import PWMAudioOut as AudioOut
+    except ImportError:
+        pass # not always supported by every board!
```

Comment:
Instead of using try/except, please check the version info instead. That'll be much clearer to read.

Reply:
I re-worked the imports again.  Tested on: Metro M4 Express with 4.1, Metro M4 Express with master, Particle Xenon with master.